modules/calib3d/src/five-point.cpp

```diff
@@ -487,6 +487,31 @@ cv::Mat cv::findEssentialMat( InputArray _points1, InputArray _points2,
     return findEssentialMat(_pointsUntistorted1, _pointsUntistorted2, cm0, method, prob, threshold, _mask);
 }
 
+int cv::recoverPose( InputArray _points1, InputArray _points2,
```

Comment:
what i dont like is the mixed use of sometimes using _ prefix for parameters and sometimes for variables. It seems to be mixed a lot in the whole code. Is there a guideline when to exactly use _ and when not to?